src/main/java/oxgame/app/VictoryChecker.java

```diff
@@ -0,0 +1,149 @@
+package oxgame.app;
+
+public class VictoryChecker {
+
+    public static boolean check(Coordinates coordinates, Board board, int winningStroke) {
+        return verifyHorizontal(coordinates,board,winningStroke) ||
+                verifyVertical(coordinates,board,winningStroke) ||
+                verifySlash(coordinates,board,winningStroke);
+    }
+    /**
+     * Verify if there is a win on HORIZONTAL.
+     * */
+    private static boolean verifyHorizontal(Coordinates coordinates, Board board, int winningStroke){
+        Symbol currentSymbol = board.getSymbol(coordinates);
+        int counter = 1;
+        counter = horizontalRight(coordinates, board, currentSymbol, counter);
+        counter = horizontalLeft(coordinates, board, currentSymbol, counter);
+        if (counter>=winningStroke) {
```

Comment:
Minor thing again, you can just return the if expression to get the same result

Reply:
Corrected. Will appear in next commit.